babel.config.js

```diff
@@ -6,33 +6,14 @@ function babelConfig(api) {
       "@babel/preset-env",
       {
         targets: {
-          browsers: ["> 1%", "last 4 versions"],
           node: 8,
         },
       },
     ],
     ["next/babel"],
   ];
 
-  const plugins = [
-    ["@babel/plugin-proposal-pipeline-operator", { proposal: "minimal" }],
-    ["lodash"],
-    [
-      "module-resolver",
-      {
-        alias: {
-          components: "./components",
-          lib: "./lib",
-          pages: "./pages",
-          styles: "./styles",
-        },
-      },
-    ],
-    ["wrap-in-js", { extensions: ["scss$"] }],
```

Comment:
Nous n'utilisons pas ce plugin: https://github.com/davibe/babel-plugin-wrap-in-js

Reply:
Nous utilisons maintenant `@zeit/next-sass`